public/js/view.js

```diff
@@ -497,7 +461,12 @@ function view(){
 				{ type: 'text', text: 'Change: ' + d.locus + ': ' + d.aao + '>' + d.aan},
 				{ type: 'link', href: locusHref, body: 'Search locus on Pubmed.' },
 				{ type: 'text', text: ''},
-				{ type: 'link', href: changeHref, body: 'Search protein sequence change on Pubmed.' }
+			    { type: 'link', href: changeHref, body: 'Search protein sequence change on Pubmed.' },
+				{ type: 'text', text: ''},
+			    { type: 'link', href: annotationsURL + '/annotations/' + geneName, body: 'View all references to this mutation'}, 
+				{ type: 'text', text: ''},
+			    { type: 'link', href: createMutationHref, body: 'Add and/or annotate a reference to this mutation.' },
+			    { type: 'text', text: ''},
```

Comment:
d3/gd3 doesn't add newlines after links within tooltips when hovering.  not sure how to fix this so this is a workaround, i'll document it.